libvcx/src/credential.rs

```diff
@@ -40,7 +40,7 @@ use v3::{
 };
 
 lazy_static! {
-    static ref HANDLE_MAP: ObjectCache<Credentials>  = Default::default();
+    static ref HANDLE_MAP: ObjectCache<Credentials> = ObjectCache::<Credentials>::new("crednetial-cache");
```

Comment:
typo

Reply:
fixed, also unified all cache names to plural